skyverge/kodiak-ui
# MWC: 7248: add styles overlay to the Dialog

# Summary 

Add `styles: { overlay: SxStyleProp }` to allow passing down overrides to the dialog down to the Overlay.

For additional context, see comments in [woosaas-site-management PR#673](https://github.com/gdcorp-commerce/woosaas-site-management/pull/673).

QA
- check in storybook
- styles can be passed down to overlay via Dialog props (isolation, background etc)
- default styles continue to work when no styles are passed in

### Issue: [MWC-7248](https://jira.godaddy.com/browse/MWC-7248)

## QA
Check in Kodiak storybook:
- Styles can be passed down to overlay via Dialog props (isolation, background etc).  For example, can add the below to `Dialog` in the existing Dialog story, and see the change when the 'Trigger' button is clicked in Kodiak storybook.

```
        styles={{
          overlay: {
            bg: 'red',
            width: '200px',
            height: '200px',
          },
        }}
```

- Default styles continue to work when no styles are passed in

## Before merge 

- [x] This PR makes the appropriate modifications to automated tests or explains why none are needed
- [x] This PR makes the appropriate modifications to documentation or explains why none are needed
- [x] This change does not impact usage or expected outputs of covered code
